packages/effect/src/internal/rateLimiter.ts

```diff
@@ -1,79 +1,30 @@
-/**
- * This is a direct port of `RateLimiter` from Rezilience
- * https://github.com/svroonland/rezilience/blob/master/rezilience/shared/src/main/scala/nl/vroste/rezilience/RateLimiter.scala
- */
-
-import * as Chunk from "../Chunk.js"
-import * as Deferred from "../Deferred.js"
 import type { DurationInput } from "../Duration.js"
 import * as Effect from "../Effect.js"
-import { pipe } from "../Function.js"
-import * as Queue from "../Queue.js"
-import * as Ref from "../Ref.js"
-import * as Stream from "../Stream.js"
-import { nextPow2 } from "./nextPow2.js"
```

Comment:
Could also delete this file again and move that back into `pubsub.ts`.

Reply:
Moved to the Number module.